instrumentation/lettuce/lettuce-5.1/javaagent/src/testCompatibility/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/LettuceCompatibilityTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.lettuce.v5_1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.lettuce.core.RedisClient;
+import io.lettuce.core.api.sync.RedisCommands;
+import io.opentelemetry.instrumentation.lettuce.v5_1.AbstractLettuceClientTest;
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+class LettuceCompatibilityTest extends AbstractLettuceClientTest {
+  @RegisterExtension
+  static final InstrumentationExtension testing = AgentInstrumentationExtension.create();
+
+  @Override
+  public InstrumentationExtension testing() {
+    return testing;
+  }
+
+  @Override
+  protected RedisClient createClient(String uri) {
+    return RedisClient.create(uri);
+  }
+
+  private static RedisCommands<String, String> syncCommands;
+
+  @BeforeAll
+  void setUp() throws UnknownHostException {
+    redisServer.start();
+
+    host = redisServer.getHost();
+    ip = InetAddress.getByName(host).getHostAddress();
+    port = redisServer.getMappedPort(6379);
+    embeddedDbUri = "redis://" + host + ":" + port + "/" + DB_INDEX;
+
+    redisClient = createClient(embeddedDbUri);
+
+    connection = redisClient.connect();
+    syncCommands = connection.sync();
+
+    syncCommands.set("TESTKEY", "TESTVAL");
+  }
+
+  @AfterAll
+  static void cleanUp() {
+    connection.close();
+    redisClient.shutdown();
+    redisServer.stop();
+  }
+
+  @Test
+  void testEmptyTrace() {
```

Comment:
perhaps `testInstrumentationDisabled` would better reflect the intent of this test?